spec/requests/querying_spec.rb

```diff
@@ -822,108 +822,6 @@ def create_vms_by_name(names)
   end
 
   describe "Querying resources" do
-    it "does not return actions if not entitled" do
```

Comment:
I think if we have good unit tests for the extracted object, we probably don't need exhaustive edge cases tested at this level 👍 I would however leave one or two general cases to show that everything works when integrated. WDYT?